code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -19,12 +19,11 @@
 
 /obj/item/storage/firstaid/regular
 	starts_with = list(
-		/obj/item/stack/medical/bruise_pack = 2,
+		/obj/item/stack/medical/bruise_pack = 3,
 		/obj/item/stack/medical/ointment = 2,
```

Comment:
```suggestion
		/obj/item/stack/medical/ointment = 3,
```

Reply:
This would overflow the kits, wouldn't it? I tried to keep the number of slots filled the same.

Reply:
I'd like the amount of bruise packs and ointment to be the same

Alternatively you can have an additional splint instead.

Reply:
I added the emergency autoinjector that spawns in people's emergency boxes instead. This PR wasn't meant to be a nerf to the power of white medkits, but instead to try and keep access to health analyzers about the same. Losing the analyzer but gaining a better ability to stabilize people before medical arrives (which is the point of these kits) seems fair.